source/_docs/upstream-updates.md

```diff
@@ -140,8 +140,8 @@ If the automated core update doesn't appear to be working, it's possible there a
 ### One-Click Updates Do Not Appear After Rewriting Git History
 Squashing and rewriting history may cause one-click updates to break, meaning updates will no longer appear on your Site Dashboard once available. Instead of using squash and rebase to clean up commits from merges occurring upstream, we recommend reviewing history locally with `git log --first-parent`. This provides the same history shown on the Site Dashboard and prevents conflicts with our one-click updates.
 
-### One-Click Update Not Available for Sites Using a Public Upstream
-Core updates for Public Upstreams (Open Atrium, Commerce Kickstart, etc.) are initiated by the maintainer, not Pantheon. Please report issues directly to the project maintainer for expected updates.
+### One-Click Update Not Available for Sites Using a Custom Upstream
+Core updates for Custom Upstreams of public distributions (Open Atrium, Commerce Kickstart, etc.) are initiated by the maintainer, not Pantheon. Please report issues directly to the project maintainer for expected updates.

```

Comment:
I'd like to stay away from 'Custom' for anything not controlled by the org for now